assets/js/modules/pagespeed-insights/components/FieldReportMetrics.js

```diff
@@ -75,30 +65,54 @@ export default function FieldReportMetrics( { data } ) {
 	const cls = ( cumulativeLayoutShift.percentile / 100 ).toFixed( 2 );
 
 	return (
-		<div>
-			<p>
-				{ __( 'Field data is useful for capturing true, real-world user experience - but has a more limited set of metrics.', 'google-site-kit' ) }
-				{ ' ' }
-				{ learnMoreLink }
-			</p>
-			<ReportMetric
-				title={ __( 'First Input Delay', 'google-site-kit' ) }
-				description={ __( 'Helps measure your user’s first impression of your site’s interactivity and responsiveness.', 'google-site-kit' ) }
-				displayValue={ `${ firstInputDelay.percentile } ms` }
-				category={ firstInputDelay.category }
-			/>
-			<ReportMetric
-				title={ __( 'Largest Contentful Paint', 'google-site-kit' ) }
-				description={ __( 'Marks the time at which the largest text or image is painted.', 'google-site-kit' ) }
-				displayValue={ `${ lcpSeconds } s` }
-				category={ largestContentfulPaint.category }
-			/>
-			<ReportMetric
-				title={ __( 'Cumulative Layout Shift', 'google-site-kit' ) }
-				description={ __( 'Measures the movement of visible elements within the viewport.', 'google-site-kit' ) }
-				displayValue={ cls }
-				category={ cumulativeLayoutShift.category }
-			/>
+		<div className="googlesitekit-pagespeed-insights-web-vitals-metrics">
+			<div className="googlesitekit-pagespeed-report__row googlesitekit-pagespeed-report__row--first">
+				<p>
+					{ __( 'Field data is useful for capturing true, real-world user experience - but has a more limited set of metrics.', 'google-site-kit' ) }
+					{ ' ' }
+					<FieldDataLearnMoreLink />
+				</p>
+			</div>
+			<table
```

Comment:
I always get excited when it's time to actually use `<table>` tags legitimately, but they still feel like a red flag at first. I wonder if that visceral response will ever go away 😅